bitnami/rabbitmq/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/bitnami-docker-rabbitmq
   - https://www.rabbitmq.com
-version: 8.21.0
+version: 8.21.1
```

Comment:
Since this change is adding a new feature, we should increase the minor version (i.e. `8.21.0` -> `8.22.0`).

Reply:
updated to 8.22.0